{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2024-12084",
      "description": "A heap-based buffer overflow flaw was found in the rsync daemon. This issue is due to improper handling of attacker-controlled checksum lengths (s2length) in the code. When MAX_DIGEST_LEN exceeds the fixed SUM_LENGTH (16 bytes), an attacker can write out of bounds in the sum2 buffer.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2024-12084",
      "reportURL": "https://github.com/themirze/cve-2024-12084",
      "timeStamp": "2025-01-21T02:13:14Z"
    }
  ],
  "trickest": [
    {
      "id": "CVE-2024-12084",
      "url": "https://github.com/rxerium/CVE-2024-12084",
      "added_at": "2025-01-19T20:27:54Z"
    }
  ],
  "nomisec": [
    {
      "cveId": "CVE-2024-12084",
      "id": 924344284,
      "name": "CVE-2024-12084",
      "full_name": "rxerium/CVE-2024-12084",
      "owner": {
        "login": "rxerium",
        "id": 59293085,
        "avatar_url": "https://avatars.githubusercontent.com/u/59293085?v=4",
        "html_url": "https://github.com/rxerium",
        "user_view_type": "public"
      },
      "html_url": "https://github.com/rxerium/CVE-2024-12084",
      "description": "A heap-based buffer overflow flaw was found in the rsync daemon. This issue is due to improper handling of attacker-controlled checksum lengths (s2length) in the code. When MAX_DIGEST_LEN exceeds the fixed SUM_LENGTH (16 bytes), an attacker can write out of bounds in the sum2 buffer.",
      "fork": false,
      "created_at": "2025-01-29T20:44:04Z",
      "updated_at": "2025-01-29T20:51:04Z",
      "pushed_at": "2025-01-29T20:50:54Z",
      "stargazers_count": 0,
      "watchers_count": 0,
      "has_discussions": false,
      "forks_count": 0,
      "allow_forking": true,
      "is_template": false,
      "web_commit_signoff_required": false,
      "topics": [
        "cyber",
        "zero-day"
      ],
      "visibility": "public",
      "forks": 0,
      "watchers": 0,
      "score": 0,
      "subscribers_count": 1
    },
    {
      "cveId": "CVE-2024-12084",
      "id": 919782397,
      "name": "cve-2024-12084",
      "full_name": "themirze/cve-2024-12084",
      "owner": {
        "login": "themirze",
        "id": 107270233,
        "avatar_url": "https://avatars.githubusercontent.com/u/107270233?v=4",
        "html_url": "https://github.com/themirze",
        "user_view_type": "public"
      },
      "html_url": "https://github.com/themirze/cve-2024-12084",
      "description": "",
      "fork": false,
      "created_at": "2025-01-21T02:13:14Z",
      "updated_at": "2025-04-11T15:19:26Z",
      "pushed_at": "2025-01-21T02:13:39Z",
      "stargazers_count": 2,
      "watchers_count": 2,
      "has_discussions": false,
      "forks_count": 2,
      "allow_forking": true,
      "is_template": false,
      "web_commit_signoff_required": false,
      "topics": [],
      "visibility": "public",
      "forks": 2,
      "watchers": 2,
      "score": 0,
      "subscribers_count": 1
    }
  ],
  "nuclei": [],
  "metasploit": [],
  "holloways": [
    {
      "cve": "CVE-2024-12084",
      "url": "https://www.cve.news/cve-2024-12084/",
      "added_at": "2025-01-28T00:00:00Z",
      "score": 1,
      "template_for": ""
    }
  ],
  "openpoc": [
    {
      "cve": "CVE-2024-12084",
      "url": "https://github.com/rxerium/CVE-2024-12084",
      "added_at": "2025-01-19T20:27:54Z",
      "score": 0.4,
      "template_for": ""
    },
    {
      "cve": "CVE-2024-12084",
      "url": "https://github.com/themirze/cve-2024-12084",
      "added_at": "2025-01-21T02:13:14Z",
      "score": 0.42,
      "template_for": ""
    },
    {
      "cve": "CVE-2024-12084",
      "url": "https://www.cve.news/cve-2024-12084/",
      "added_at": "2025-01-28T00:00:00Z",
      "score": 1,
      "template_for": ""
    }
  ]
}
